reconcile/statuspage/integrations/maintenances.py

```diff
@@ -46,13 +47,15 @@ def reconcile(
     def run(self, dry_run: bool = False) -> None:
         binding_state = get_binding_state(self.name, self.secret_reader)
         pages = get_status_pages()
+        now = datetime.now(timezone.utc)
 
         error = False
         for p in pages:
             try:
                 desired_state = [
                     StatusMaintenance.init_from_maintenance(m)
                     for m in p.maintenances or []
+                    if datetime.fromisoformat(m.scheduled_start) > now
```

Comment:
this is the important part of the PR, where we only manage maintenances that have not yet started (and matching qontract-cli with the same behavior)